### DISCUSSION TITLE
제페토 3D 아이템 헤어 업로드하면 헤어 색깔이 이상해요ㅠ

### DISCUSSION THREAD

**Discussion by phy9051** [naverz/zepeto-studio-kor#637] [5. Q&A]
월드 관련 질문은 아니지만 혹시 아시는 분들 계시면 응답 바랍니다!

제페토 아이템 헤어 컬러링해서 유니티 변환할 때까지,
입힌 컬러 거의 그대로 구현되는데
제페토 스튜디오에 업로드하면 칙칙하게 바뀌어요.

그런데 제페토 앱에서 보면 다른 헤어 아이템들 색상이 매우 다양하게 잘 되어있는데
혹시 헤어 색 조절하는 방법이 따로 있을까요??

**Comment by x4cf** [comment]
헤어 색깔은 아이템 구매 이후 해어컬러 변경에서 하실 수 있어요

**Reply by phy9051** [reply]
감사합니다ㅠ!!

**Comment by beanyh** [comment]
- 유니티에서 material 네이밍 뒤에 "nocolor"를 추가해야 합니다.

**Reply by phy9051** [reply]
감사합니다!!!!

**Comment by Zepeto-TechEvangelist** [comment]
가장 도움이 된 답변을 Mark as answer로 채택 부탁 드립니다. 🥰

**Reply by phy9051** [reply]
네 알겠습니다!
